Bio/PDB/parse_pdb_header.py

```diff
@@ -166,7 +182,8 @@ def parse_pdb_header(infile):
             record_type = line[0:6]
             if record_type in ("ATOM  ", "HETATM", "MODEL "):
                 break
-            header.append(line)
+            else:
```

Comment:
No need for the else here. The for loop will run until it his one of the record_types above and then will break (stop the for-loop).